### DISCUSSION TITLE
Is there an API endpoint for trending posts?

### DISCUSSION THREAD

**Discussion by goulvenclech** [mastodon/mastodon#20169] [Q&A] [answered]
Hi everyone, 

I'm working on a channel `Mastodon` in my Discord server, fueled by our instance's trending posts. Unfortunately, I couldn't put my hand on the dedicated endpoint in the documentation, and I don't have the time to frisk into the source code right now...

Is there an undocumented endpoint? Or is this a missing feature I could work on? 

Thanks for your help.

**Comment by trwnh** [comment, marked as answer]
`GET https://mastodon.example/api/v1/trends/statuses`

It's not live yet on the docs site, but it is part of https://github.com/mastodon/documentation/pull/991

**Reply by goulvenclech** [reply]
Thanks @trwnh for your answer! I can't wait for your PR to be merged :)
